beginner_source/flava_finetuning_tutorial.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+TorchMultimodal Tutorial : Finetuning FLAVA
+=======================
+"""
+
+######################################################################
+# Multimodal AI has recently become very popular owing to its ubiquitous
+# nature, from use cases like image captioning and visual search to more
+# recent applications like image generation from text. **TorchMultimodal
+# is a library powered by Pytorch consisting of building blocks and end to
+# end examples, aiming to enable and accelerate research in
+# multimodality**.
+# 
+# In this tutorial, we will demonstrate how to use a **pretrained SoTA
+# model called** `FLAVA <https://arxiv.org/pdf/2112.04482.pdf>`__ **from
+# TorchMultimodal library to finetune on a multimodal task i.e. visual
+# question answering** (VQA).
+# 
+
+
+######################################################################
+# Installations
+# 
+# We will use TextVQA dataset from HuggingFace for this
+# tutorial. So we install datasets in addition to TorchMultimodal
+# 
+
+# TODO: replace with install from pip when binary is ready
+!git clone https://github.com/facebookresearch/multimodal.git
+!pip install -r multimodal/requirements.txt
+import os
+import sys 
+sys.path.append(os.path.join(os.getcwd(),"multimodal"))
+sys.path.append(os.getcwd())
+!pip install datasets
+!pip install transformers
```

Comment:
already in requirements.txt

Reply:
Yes, but we want it to be present in the notebook